operations/onboarding.md

```diff
@@ -0,0 +1,63 @@
+# Getting started and onboarding
+
+This is a short guide for welcoming new team members to the 2i2c team.
+Its goal is to give others an idea for where to look for information and how to familiarize themselves with our structure and processes.
+
+:::{admonition} Don't try to learn all of this at once!
+2i2c is a distributed organization, which means that we rely **heavily** on documentation.
+Don't worry about figuring it all out immediately - this guide will try to focus on the most important parts to get started.
+:::
+
+## The onboarding process
+
+First off, here's the process we follow when onboarding new team members.
+
+When a new team member joins, follow these two steps first:
+
+1. **Identify an Onboarding Champion**. This is somebody that will help guide the onboarding process for our new team member.
+   Their job is to carry out the process described in the onboarding issue template.
+2. **Open an onboarding issue in the Team Compass**.
```

Comment:
As this document appears to address an existing team member, perhaps in this section add a link to the `new-team-member` issue template.

https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/new-team-member.md

I also think you're missing a "send a welcome email" that should include asking for their github account (and giving a link to creating one for non-end team members who might not have one already).